scout/adapter/mongo/omics_variant.py

```diff
@@ -82,8 +82,8 @@ def set_samples(self, case_obj: dict, omics_model: dict):
         )
 
         sample = {
-            "sample_id": match.get("omics_sample_id", match["individual_id"]),
-            "display_name": match["display_name"],
+            "sample_id": match["individual_id"],
+            "display_name": match.get("omics_sample_id") or omics_model["sample_id"],
```

Comment:
Here one can display the name provided as sampleID in the TSV file tho

Reply:
Yes, precisely.

Reply:
Or wait, no, stop leading me on! 😂 That is the CG internal ID. It will have no meaning to the customers - we are much better off displaying the matched sample id.

Reply:
display_name to display_name, sample_id to sample_id. I can update if you want. But there is still at least one more issue hiding..

Reply:
Sure, push. What's hiding?

Reply:
It worked right before merging on hasta-stage. I've rechecked the case in db, and it does not seem to have used any fallback, which makes me think there is a compensatory "error" later in the chain.